graphwalker-io/src/main/java/org/graphwalker/io/factory/yed/YEdContextFactory.java

```diff
@@ -269,13 +279,25 @@ private Vertex addVertices(Model model, GraphmlDocument document, Map<String, Ve
             workQueue.addAll(Arrays.asList(subgraph.getNodeArray()));
           }
         } else {
-          String description = "";
+          Vertex vertex = new Vertex();
+          for(Map.Entry<String, KeyType> entry : propKeys.entrySet()) {
```

Comment:
Try follow the syntax that is used in the rest of the file, e.g spaces `for(` => `for (`